libs/core/src/lib/input-group/input-group.component.html

```diff
@@ -26,7 +26,7 @@
                 fd-button
                 class="fd-input-group__button"
                 type="button"
-                [fdType]="inShellbar ? 'standard' : 'transparent'"
+                [fdType]="'transparent'"
```

Comment:
`fdType="transparent"`

Reply:
It search for `transparent`  property in component. `[fdType]="'transparent'"` will pass as a value to `fdType`.

Reply:
it won't. if you want to pass `transparent` value from the component - use it without `''`.